package.json

```diff
@@ -127,7 +127,7 @@
     }
   },
   "prettier": {
-    "printWidth": 100,
+    "printWidth": 120,
```

Comment:
Function declarations were getting collapsed into an unreadable mess when return types were added. This made many cases look nicer.